openpower/package/ppe42-gcc/0001-2016-02-19-Jakub-Jelinek-jakub-redhat.com.patch

```diff
@@ -52,7 +52,7 @@ index 2784edc..8d7f015 100644
 -#endif
 -static unsigned int hash (const char *, unsigned int);
 -#ifdef __GNUC__
--__inline
+-//__inline
```

Comment:
why this change?

Isn't this a backported fix? Why would it suddenly change?

Reply:
Because I also made changes in this area in a previous version